inst/tinytest/testRcppInterfaceExporter/src/RcppExports.cpp

```diff
@@ -43,7 +43,9 @@ RcppExport SEXP _testRcppInterfaceExporter_test_cpp_interface(SEXP xSEXP, SEXP f
     if (rcpp_isError_gen) {
         SEXP rcpp_msgSEXP_gen = Rf_asChar(rcpp_result_gen);
         UNPROTECT(1);
+        #undef Rf_error
         Rf_error("%s", CHAR(rcpp_msgSEXP_gen));
+        #include <Rcpp/macros/mask.h>
```

Comment:
Eww. #include as code path?  Should we not just carry the definition over here?  (I know, code duplication and all that...)  But still.

Reply:
There is a `#pragma` to push/pop definitions, which is much cleaner. But since we cannot use that on CRAN if I'm not mistaken, then this is the cleanest solution I could think off without, yeap, inserting the same definition again, which will end up in many packages, because this is `RcppExports.cpp`. I feel we should control this more tightly.

Reply:
Mmmh, I think I have a better solution.

Reply:
I pondered this a bit more and I think I can live with it too.  I see what you're doing and where you are coming from.

But hey, if there is something better still...